### PR TITLE
reenable "-exchange=1"; locktime=0

### PR DESCRIPTION
**Pull request by Alrighttt**
Some services are experiencing issues with their rewards calculations. This will disable them altogether if they set `-exchange=1` while starting their daemon. 

@dimxy @jmjatlanta @DeckerSU @ca333 

edit: It should be noted that `IS_MODE_EXCHANGEWALLET` is an already established feature. Please consider this in your review.

### CODE REVIEW

**Review by DeckerSU — DISMISSED** [review]
LGTM.

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -3795,6 +3795,8 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
     else
         txNew.nLockTime = (uint32_t)chainActive.Tip()->GetMedianTimePast();
 
+    if (IS_MODE_EXCHANGEWALLET && ASSETCHAINS_SYMBOL[0] == 0) txNew.nLockTime = 0;
```
Comment (jmjatlanta) [inline src/wallet/wallet.cpp:3798]:
```suggestion
    if (IS_MODE_EXCHANGEWALLET && ASSETCHAINS_SYMBOL[0] == 0) 
        txNew.nLockTime = 0;
    else
    {
        if ( !komodo_hardfork_active((uint32_t)chainActive.LastTip()->nTime) )
            txNew.nLockTime = (uint32_t)chainActive.LastTip()->nTime + 1; // set to a time close to now
        else
            txNew.nLockTime = (uint32_t)chainActive.Tip()->GetMedianTimePast();
    }
```

**Review by jmjatlanta — APPROVED** [review]
Minor optimization below, but LGTM.

**Comment by DeckerSU** [comment]
May be also good will be to init `IS_MODE_EXCHANGEWALLET` explicitly in `komodo_globals.h`:
https://github.com/Alrighttt/komodo/blob/22c394515b4038542ca47879bae2206e041b0f8d/src/komodo_globals.h#L59
like:
```
bool IS_MODE_EXCHANGEWALLET = false;
```
To be sure that if somebody further will remove the following line again:
```
IS_MODE_EXCHANGEWALLET = GetBoolArg("-exchange", false);
```
The value of `IS_MODE_EXCHANGEWALLET` will not leaved uninitialized.

**Review by DeckerSU — APPROVED** [review]
LGTM after changes.

**Review by ca333 — APPROVED** [review]

**Pull request merged**